lib/pry/input_completer.rb

```diff
@@ -239,4 +227,30 @@ def build_path(input)
     end
     return path, input
   end
+  
+    private
+
+    def self.fetch_obj_space_collection
+      (self.obj_spase_mutex ||= Mutex.new).synchronize do
+        count = ObjectSpace.each_object(Module).count
```

Comment:
It is not a bug. I wanted to recalc ObjectSpace if Module count changes. Actually, it is not the best solution, bun much better, than preload everything.